config.js

```diff
@@ -98,5 +98,17 @@ module.exports = {
       name: 'provider',
       url: 'https://provider.mainnet.oceanprotocol.com/spec'
     }
-  ]
+  ],
+  v3Versions: {
+    'ocean.js': 'v0.19.0',
+    'ocean.py': 'v0.8.5',
```

Comment:
On ocean.py there is no "Reference". We can point to: https://deploy-preview-973--docs-oceanprotocol-v3.netlify.app/references/read-the-docs/ocean-py/
![Screenshot 2022-04-26 at 13 06 34](https://user-images.githubusercontent.com/84312885/165276815-1822b6fe-15ea-47c9-97ce-dbd3e80acc07.png)